pom.xml

```diff
@@ -2001,12 +2001,6 @@
                 <version>2.6.2</version>
             </dependency>
 
-            <dependency>
```

Comment:
why did this get added to the presto-root pom in the original commit? @tdcmeehan @infvg

Reply:
Since I upgraded commons-compress to 1.26.2, I was getting a require upper bounds error relating commons-lang3 in presto-product-tests:
```
Require upper bound dependencies error for org.apache.commons:commons-lang3:3.3.2 paths to dependency are:
+-com.facebook.presto:presto-product-tests:0.290-SNAPSHOT
  +-io.prestodb.tempto:tempto-core:1.53
    +-org.apache.commons:commons-lang3:3.3.2
and
+-com.facebook.presto:presto-product-tests:0.290-SNAPSHOT
  +-org.apache.avro:avro:1.11.4
    +-org.apache.commons:commons-compress:1.26.2
      +-org.apache.commons:commons-lang3:3.14.0
```
I added chose to upgrade commons-lang3 too to resolve this issue and resolve future conflicts. I saw it wasn't breaking any functionality so I felt it was fine.